operations/mimir-mixin/dashboards/block-builder.libsonnet

```
@@ -0,0 +1,95 @@
+local filename = 'mimir-block-builder.json';
+
+(import 'dashboard-utils.libsonnet') +
+{
+  [filename]:
+    assert std.md5(filename) == 'c565e768420d79d0a632b3135d47cb30' : 'UID of the dashboard has changed, please update references to dashboard.';
+    ($.dashboard('Block-builder') + { uid: std.md5(filename) })
+    .addClusterSelectorTemplates()
+    .addRow(
+      $.row('Summary')
+      .addPanel(
+        $.timeseriesPanel('Overall consumption / sec') +
+        $.panelDescription(
+          'Overall consumption / sec',
+          'Overview of per-second rate of records consumed from Kafka.',
+        ) +
+        $.queryPanel(
+          [
+            'sum (rate(cortex_ingest_storage_reader_fetch_records_total{%(job)s}[$__rate_interval]))' % { job: $.jobMatcher($._config.job_names.block_builder) },
+            'sum (rate(cortex_ingest_storage_reader_read_errors_total{%(job)s}[$__rate_interval]))' % { job: $.jobMatcher($._config.job_names.block_builder) },
+          ],
+          [
+            'successful',
+            'read errors',
+          ],
+        ) +
+        $.stack,
+      )
+      .addPanel(
+        $.timeseriesPanel('Per-instance consumption / sec') +
```

[nit] Suggestion to keep it consistent with other dashboards:

```suggestion
        $.timeseriesPanel('Per pod Kafka fetched records / sec') +
```